tests/__init__.py

```diff
@@ -0,0 +1,112 @@
+import os
+import unittest
+
+import responses
+
+import roboflow
+from roboflow.config import API_URL
+
+ROBOFLOW_API_KEY = "my-test-app"
+WORKSPACE_NAME = "my-workspace"
+PROJECT_NAME = "test-project"
+PROJECT_VERSION = "1"
+
+
+def make_orderer():
+    order = {}
+
+    def ordered(f):
+        order[f.__name__] = len(order)
+        return f
+
+    def compare(a, b):
+        if a not in order or b not in order:
+            return 1
+
+        return [1, -1][order[a] < order[b]]
+
+    return ordered, compare
+
+
+ordered, compare = make_orderer()
+
+unittest.defaultTestLoader.sortTestMethodsUsing = compare
```

Comment:
this test ordering stuff was just moved over from the old `test.py` file... with a slight modification to `compare` to handle tests not decorated with `ordered`.